### PR TITLE
Add image on notification support

### PR DESCRIPTION
**Pull request by chliangGoogle**
Add extensionHelper class that can be used inside notification extension to render image.

### CODE REVIEW

File: Firebase/Messaging/FIRMessagingExtensionHelper.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRMessagingExtensionHelper : NSObject
+
+/// Call this API to complete your notification content modification. If you like to
+/// overwrite some properties of the content instead of using the default payload,
+/// make sure to make your customized motification to the content before passing it to
+/// this call.
+- (void)populateNotificationContent:(UNMutableNotificationContent *)content
+                 withContentHandler:(void (^)(UNNotificationContent *_Nonnull))contentHandler;
```
Comment (paulb777) [inline Firebase/Messaging/FIRMessagingExtensionHelper.h]: Looks like UNNotificationContent is not available on tvOS

File: Firebase/Messaging/FIRMessagingExtensionHelper.m [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            return;
+          }
+
+          attachment = [UNNotificationAttachment attachmentWithIdentifier:@""
+                                                                      URL:localURL
+                                                                  options:nil
+                                                                    error:&error];
+          if (error) {
+            FIRMessagingLoggerError(kFIRMessagingServiceExtensionImageNotAttached,
+                                    @"Failed to attach the image to attachments, error: %@\n",
```
Comment (paulb777) [inline Firebase/Messaging/FIRMessagingExtensionHelper.m]: Should message be "Failed to create attachment"?  Should the localURL be part of the error message?

Comment (chliangGoogle) [reply Firebase/Messaging/FIRMessagingExtensionHelper.m]: Good call.

File: Example/Firebase.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1914,6 +1919,9 @@
 		6003F581195388D10070C39A = {
 			isa = PBXGroup;
 			children = (
+				518855102231E09300CA4141 /* FIRMessagingExtensionHelperTest.m */,
+				5188550F2231E04C00CA4141 /* FIRMessagingClientTest.m */,
+				5188550D2231E02300CA4141 /* FIRMessagingExtensionHelperTest.m */,
```
Comment (renkelvin) [inline Example/Firebase.xcodeproj/project.pbxproj]: These 3 files are placed in the root directory of the Firebase Example project. Please move them into Messaging/.

**Pull request merged**